sdk/edgeorder/Azure.ResourceManager.EdgeOrder/src/Custom/EdgeOrderItemCollection.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Autorest.CSharp.Core;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager.Resources;
+
+[assembly: CodeGenSuppressType(nameof(IEnumerator))]
```

Comment:
The new list operation have new `int top` parameter.
We added the old `GetAll` method back to the custom code, but will it casue ambiguous call in the `IEnumerator` methods between those two `GetAll()`.
How to suppress them? Or how to avoid this?

Reply:
Fixed